pipeline_dp/dp_computations.py

```diff
@@ -215,39 +193,40 @@ def compute_dp_sum(sum: float, dp_params: MeanVarParams):
         ValueError: The noise kind is invalid.
     """
     l0_sensitivity = dp_params.l0_sensitivity()
-    linf_sensitivity = dp_params.linf_sensitivity(pipeline_dp.Metrics.SUM)
+    linf_sensitivity = dp_params.max_contributions_per_partition * max(
+        dp_params.low, dp_params.high)
 
     return _add_random_noise(sum, dp_params.eps, dp_params.delta,
                              l0_sensitivity, linf_sensitivity,
                              dp_params.noise_kind)
 
 
-def _compute_mean(count: float, dp_count: float, sum: float, eps: float,
-                  delta: float, l0_sensitivity: float,
-                  metric: pipeline_dp.Metrics, dp_params: MeanVarParams):
+def _compute_mean(count: float, dp_count: float, sum: float, low: float,
+                  high: float, eps: float, delta: float, l0_sensitivity: float,
+                  linf_sensitivity: float, noise_kind: pipeline_dp.NoiseKind):
     """Helper function to compute the DP mean of a raw sum using the DP count.
 
     Args:
         count: Non-DP count.
         dp_count: DP count.
         sum: Non-DP sum.
+        low, high: The lowest/highest contribution.
         eps, delta: The budget allocated.
         l0_sensitivity: The L0 sensitivity.
-        metric: The metric used at computing the Linf sensitivity.
-        dp_params: The parameters used at computing the noise.
+        linf_sensitivity: The Linf sensitivity.
```

Comment:
Why not pass in max_contributions_per_partition instead and compute linf_sensitivity in this method? That way the caller doesn't have to account for the normalization.

Reply:
Sure, sounds good since the sensitivity is also computed similarly.